test/index.js

```diff
@@ -54,192 +55,244 @@ describe("getEnvironmentVariables", () => {
 		});
 	});
 
-	it("returns correct variables", () => {
-		fs.existsSync = (filePath) => {
-			return filePath !== etcPaths;
-		};
+	describe("when Node.js is used", () => {
+		it("prints warning that cannot get environment variables via node when spawn of child process with node arg fails", () => {
+			fs.existsSync = (filePath) => {
+				return filePath !== etcPaths;
+			};
 
-		const expectedVariables = {
-			"VAR1": "1",
-			"VAR2": "2",
-			"VAR4": "1=2=3=4",
-			"VAR5": "   1 2   3 ",
-			"VAR6": "1 2 3"
-		};
+			const execSyncCommands = [];
+			childProcess.execSync = (command) => {
+				execSyncCommands.push(command);
+				if (command.indexOf("node") !== -1) {
+					throw new Error("Unable to find node");
+				}
 
-		let childProcess = require("child_process");
-		childProcess.execSync = (command) => {
-			return _.map(expectedVariables, (value, key) => {
-				return `${key}=${value}`;
-			}).join('\n');
-		};
+				return _.map(defaultExpectedVariables, (value, key) => {
```

Comment:
You can shorten the fat arrow function passed as a second argument to `_.map` here